assets/component-card.css

```diff
@@ -362,3 +367,119 @@
 .card-article-info {
   margin-top: 1rem;
 }
+
+/* Card Shapes */
+
+.card--shape--shape .card__content {
+  padding-top: 0;
+}
+
+.card--shape.card--standard .card__inner {
+  background-color: transparent;
+}
+
+.card--shape--oval.card--media .card__media {
+  clip-path: ellipse(45% 45% at 50% 50%);
+}
+
+.card--shape--arch.card--media .card__media {
+  clip-path: url("#image-shape--arch");
+}
+
+.card--shape--parallelogram.card--media .card__media {
+  clip-path: polygon(15% 0, 100% 0%, 85% 100%, 0% 100%);
+}
+
+.card--shape--diamond.card--media .card__media {
+  clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+}
+
+.card--shape--chevronright.card--media .card__media {
+  clip-path: polygon(85% 0%, 100% 50%, 85% 100%, 0% 100%, 15% 50%, 0% 0%);
+}
```

Comment:
Can we prevent having to duplicate the paths shared with image with text? Particularly if we'll likely add this feature in other places in the future.

Simply capturing the property values in variables in base.css would be a slight improvement
```
/* base.css */
:root {
  --oval-path: ellipse(45% 45% at 50% 50%);
  --arch-path: url("#image-shape--arch");
  --diamond-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
  ...
}

/* comp-card.css or image-with-text.css */
.card--shape--diamond.card--media .card__media {
  clip-path: var(--diamond-path);
}
```

But ideally I'd love to assign the shape styles to their own generic classes and apply those classes where need in each section. This would allow for easy re-use and better maintainability/consistency.
```
/* base.css */
.shape--diamond {
  clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
}
...

/* card-product.liquid */
<div class="card__media shape--diamond">
```

Reply:
The shared class-based approach is a good one. I've applied it in 5f45b4b. Thanks for all your suggestions! This should be ready for another look.